src/components/List/List.component.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Card from '../Card';
+import { useVideoContext, useAuthContext } from '../../utils/store/context';
+import { Containerlist } from './List.styled';
+
+const List = ({ dataList, handleSelect }) => {
+  const { favorites, saveFavorite, updateFavorite } = useVideoContext();
+  const { user } = useAuthContext();
+
+  const handleIsFavorite = (videoId) => {
+    const fav = favorites.some((video) => video.id.videoId === videoId);
+    return fav;
+  };
```

Comment:
this one can be something like:

```suggestion
  const handleIsFavorite = videoId => (
    favorites.some(video => video.id.videoId === videoId)
  );
```